src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/HmppsDomainEventEmitter.kt

```diff
@@ -188,7 +187,6 @@ data class PrisonerReceivedEvent(
   override val nomsNumber: String,
   val reason: String,
   val prisonId: String,
-  val fromPrisonId: String?
 ) : PrisonerAdditionalInfo()
```

Comment:
decided to drop this; to hard to get this right so wait until someone really needs it